spec/requests/topics_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+RSpec.describe TopicsController do
+  ADDITIONAL_QUERY_LIMIT = 6
```

Comment:
Do the additional queries add up to 6 or is this set to 6 so that there's some extra room? If this number is precise and consistent across test runs, I'd prefer using that in tests so that an extra query can be more quickly detected when adding other features or making unrelated refactors to the plugin.

Reply:
Yeah fair. I think they actually add up to 5 for first post and 6 for full topic. I'll revisit this specific point when I take a second look at topic performance (after categories).